src/lava_callback.py

```diff
@@ -600,7 +600,8 @@ async def checkout(data: ManualCheckout, request: Request,
                 "tree": treename,
                 "branch": branch,
                 "commit": commit,
-                "url": treeurl
+                "url": treeurl,
+                "tip_of_branch": False
```

Comment:
Custom checkout can also have a commit from the tip of the branch. Why is it set `False`?

Reply:
We cannot trust user to indicate if it is TOT, but to test it separately on our side will need additional logic to implement (or fix existing one). 
We might check it later, as it is not critical right now, custom checkout events should not be sent to kcidb anyway (yet).

Reply:
I don't think we rely much on the fact if the commit is at the tip of branch or not. We are just using that data to send to KCIDB.
I think we can set its value from user's input and set it to `None` if not provided.

Reply:
It worked fine, your code updated it to true, if it is tipoftree, so we can leave as is